src/workloads/query/LimitSkip.yml

```diff
@@ -0,0 +1,142 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload measures performance for $limit and $skip when different constants are supplied as
+  arguments.
+
+Keywords:
+- limit
+- skip
+- cache
+- parametrization
+- classic
+- sbe
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e6
+  Threads: &Threads 1
+  MaxPhases: &MaxPhases 6
+  LimitSkipNumber: &limitSkipNum {^Inc: {start: 1, step: 100}}
+
+ActorTemplates:
+- TemplateName: RunAggCommand
+  Config:
+    Name: {^Parameter: {Name: "name", Default: "RunAggCommand"}}
+    Type: CrudActor
+    Database: *Database
+    Collection: *Collection
+    Threads: 4
+    Phases:
+      OnlyActiveInPhases:
+        Active: {^Parameter: {Name: "active", Default: [0]}}
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Repeat: 1000
+          Database: *Database
+          Collection: *Collection
+          Operations:
+          - OperationMetricsName: sbe
```

Comment:
Do we really need it? I think we can just remove this parameter and have auto-generated metric names from actor name.